spec/unit/classes/server/config_spec.rb

```diff
@@ -34,8 +34,9 @@
           class { 'postgresql::globals':
             manage_package_repo => true,
             version => '9.4',
-          }->
-          class { 'postgresql::server': }
+            before => Class['Postgresql::Server'],
+          }
+          include postgresql::server
```

Comment:
Those are probably all related to https://tickets.puppetlabs.com/browse/PUP-6608 and should be reverted before merging. 4.6.1 release should be out this week.